mne/minimum_norm/time_frequency.py

```diff
@@ -335,11 +335,16 @@ def source_induced_power(epochs, inverse_operator, frequencies, label=None,
         and if b is None then b is set to the end of the interval.
         If baseline is equal ot (None, None) all the time
         interval is used.
-    baseline_mode : None | 'logratio' | 'zscore'
+    baseline_mode : None | 'ratio' | 'zscore' | 'mean' | 'percent' | 'logratio' | 'zlogratio'
```

Comment:
pep8 fails because this line is too long:
```
Running flake8
mne/minimum_norm/time_frequency.py:338:80: E501 line too long (93 > 79 characters)
```
but there is some spell that makes pep8 ignore a line, @Eric89GXL will tell you :)